src/components/NavBar/Navbar.tsx

```diff
@@ -0,0 +1,28 @@
+'use client';
+import React, { useState, useEffect } from "react";
+import styles from './styles.module.css';
+import Image from "next/image";
+import { useRouter } from 'next/router';
+import Link from "next/link";
+
+function NavBar() {
+    return (
+        <header className={styles.Header}>
+            <div className={styles.color}>
+                <div className={styles.allHeader}>
+                    <nav>
+                        <ul className={styles.navList}>
```

Comment:
Cambiar la fuente a la definida en el look and feel, antes de volver a abrir el PR, favor enviar pantallazo de como se ve con esta implementación